packages/server/controllers/index.js

```diff
@@ -17,6 +17,18 @@ const transcribeHandler = async (req, res) => {
   });
 };
 
+
+const completionHandler = async (text) => {
+  // const input = "Talk me about seafish, but first tell me what day is today."
+  
+  const completion = await openAi.createChatCompletion({
+    model: "gpt-3.5-turbo",
+    messages: [{role: "user", content: text}],
+  });
```

Comment:
There is no error handling in case it occurs, it's better to use the methods provided by promises. Please add .then and .catch.